stocks/stocks.py

```diff
@@ -1,7 +1,155 @@
-from flask import Blueprint
+import requests
+import os
+import json
+import sqlite3 as sql
+import config
 
-stocks_bp = Blueprint('stocks_bp', __name__)
+from flask import Blueprint, request
+from dotenv import load_dotenv
 
-@stocks_bp.route('/')
+
+load_dotenv()
+
+FINNHUB_TOKEN = os.getenv("FINNHUB_TOKEN")
+
+STOCK_QUOTE_URL = "https://finnhub.io/api/v1/quote?token={token}&symbol={symbol}"
+STOCK_PROFILE_URL = "https://finnhub.io/api/v1/stock/profile2?token={token}&symbol={symbol}"
+
+stocks_bp = Blueprint("stocks_bp", __name__)
+
+@stocks_bp.route("/example")
 def index():
-    return "This is an example stock app"
\ No newline at end of file
+    return "This is an example stock app"
+
+def get_stock_quote(symbol):
+    """
+    Get real-time quote data for a stock given symbol
+    """
+    r = requests.get(STOCK_QUOTE_URL.format(token=FINNHUB_TOKEN, symbol=symbol))
+    response = r.json()
+    return response
+    """
+        {
+            "c": 261.74,
+            "h": 263.31,
+            "l": 260.68,
+            "o": 261.07,
+            "pc": 259.45,
+            "t": 1582641000 
+        }
+    """
+
+def validate_stock_symbol(symbol):
+    """
+    Make sure symbol is trackable
+    """
+    r = requests.get(STOCK_PROFILE_URL.format(token=FINNHUB_TOKEN, symbol=symbol))
+    response = r.json()
+    if response: 
+        return response.get("name")
+    return False
+
+
+
+def calculate_percent_change(response, average_cost):
+    """
+    Calculate percent change in stock compared to average cost
+    """
+    current_price = response["c"]
+    difference = current_price - average_cost
+    percent_difference = round(difference / average_cost * 100, 2)
+    return percent_difference
+    
+
+def insert_stock_tracker(stock_id, average_cost, percent, increase, decrease):
+    """
+    Insert tracked stock average cost, percent, increase, decrease into database
+    """
+    con = sql.connect(config.DATABASE) 
```

Comment:
I think you should synchronize your db conn creation. Some use `with`, some don't

Also, it's a pretty good idea to create a db helper file/class too to handle db operations